generator/sequence_transform_config_sample.json

```diff
@@ -1,10 +1,11 @@
 {
-    "rules_file_name": "./magic.txt",
-    "magic_chars": "☆✵★✪",
-    "wordbreak_char": "⎵",
-    "output_func_chars": "↻⇑",
-    "comment_str": "//",
+    "rules_file_name": "sequence_transform_dict.txt",
+    "sequence_token_symbols": "👆👍★✪",
+    "wordbreak_symbol": "⎵",
+    "output_func_symbols": "↻⇑",
+    "comment_str": "#",
     "separator_str": "⇒",
-    "quiet": true,
-    "record_rule_usage": true
+    "sequence_tokens_ascii": "*@$#",
```

Comment:
Hmm, it's a bit weird to have `sequence_token_symbols` and `sequence_tokens_ascii`. Maybe `sequence_token_ascii` ?

Reply:
Maybe `sequence_token_ascii_chars`?